build.gradle.kts

```diff
@@ -11,8 +11,8 @@ plugins {
     id("net.researchgate.release")
 }
 
-val releaseVersion: String = (project.version) as String
-val isReleaseVersion = !releaseVersion.endsWith("SNAPSHOT")
+val lionwebVersion: String by project
```

Comment:
I would've expected `lionWebVersion`: that would be consistent with camel-casing elsewhere.

Also: I find `releaseVersion` (or `version`) a clearer name, because now “LionWeb version” could mean “serialization format version” (although that doesn't make much sense) as well